ui/app/AppLayouts/Profile/helpers/ContactDetails.qml

```diff
@@ -0,0 +1,113 @@
+import QtQuick 2.15
+
+import StatusQ 0.1
+import StatusQ.Core.Utils 0.1
+
+import AppLayouts.Profile.stores 1.0
+
+import utils 1.0
+
+QObject {
+    id: root
+
+    required property ContactsStore contactsStore
+    required property ProfileStore profileStore
+    required property string publicKey
+
+    readonly property alias loading: d.loading
+
+    // model properties
+    readonly property string displayName: d.contactDetails.displayName ?? ""
+    readonly property string ensName: d.contactDetails.ensName ?? ""
+    readonly property bool ensVerified: d.contactDetails.isEnsVerified ?? false
```

Comment:
```suggestion
    readonly property bool isEnsVerified: d.contactDetails.isEnsVerified ?? false
```

wdyt? to go along with other `isXXX` bool props below, and to also match the backend name

Reply:
Fair enough